git-ftp

```diff
@@ -996,12 +997,24 @@ download_remote_updates () {
 		mirror_options="$mirror_options -v"
 	fi
 
+	include=""
+	ignoreall=""
+	# Mirror only the files from the diff between the FTP commit and the current branch/commit
+	if [ $PULL_CHANGED_ONLY -eq 1 ] && [ ! -z $CURRENT_BRANCH ]; then
```

Comment:
It makes total sense.
And it reminded me I forgot to declare the PULL_CHANGED_ONLY var :)